rl/agents/sarsa.py

```diff
@@ -0,0 +1,471 @@
+from rl.agents.dqn import DQNAgent
+import warnings
+from copy import deepcopy
+
+import numpy as np
+from keras.callbacks import History
+
+from rl.callbacks import TestLogger, TrainEpisodeLogger, TrainIntervalLogger, Visualizer, CallbackList
+from rl.agents.dqn import mean_q
+from rl.util import huber_loss
+from keras.layers import Input, Lambda
+from keras.models import Model
+import keras.backend as K
+
+
+class Sarsa(DQNAgent):
+    def __init__(self, model, *args, **kwargs):
+        super(Sarsa, self).__init__(model, *args, **kwargs)
+
+        self.state0 = None
+        self.action0 = None
+        self.nextAction = None
```

Comment:
Pretty nitpicky detail but please use `next_action` instead to be consistent with the rest of the project. Applies equally for all other variable names.